modules/support/client/components/BlockMemberLink.component.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * A link to beport member
+ * @param {string} username - the username of the user to report
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import '@/config/client/i18n';
+import { useTranslation } from 'react-i18next';
+
+import * as api from '../api/block.api';
+
+export default function BlockMemberLink({ username }) {
+  const { t } = useTranslation('support');
```

Comment:
In https://github.com/Trustroots/trustroots/pull/2121/files#r584354289 I suggested moving to `user`module instead, so just noting that translation namespace would need changing in that case as well.